mysql-test/suite/innodb_fts/t/foreign_key_update.test

```diff
@@ -32,3 +32,16 @@ SELECT * FROM t1_fk WHERE MATCH(a) AGAINST('database');
 
 DROP TABLE t1_fk;
 DROP TABLE t1;
+
+--echo #
+--echo # MDEV-32346  Assertion failure sym_node->table != NULL
+--echo #	in pars_retrieve_table_def on UPDATE
+--echo #
+CREATE TABLE t1 (a INT PRIMARY KEY) ENGINE=InnoDB;
+CREATE TABLE t2 (a INT, b TEXT, FOREIGN KEY(a) REFERENCES t1(a),
+		 FULLTEXT (b))ENGINE=InnoDB;
+INSERT INTO t1 SET a=1;
+ALTER TABLE t2 DISCARD TABLESPACE;
+--error ER_ROW_IS_REFERENCED_2
+UPDATE t1 SET a=2;
```

Comment:
I think that it is wrong to allow `ALTER TABLE…DISCARD TABLESPACE` when `FOREIGN KEY` constraints exist and `foreign_key_checks` are enabled. This should be treated in a similar way to `DROP TABLE` or `TRUNCATE TABLE`. Please file a separate bug for that.

Reply:
https://jira.mariadb.org/browse/MDEV-33612